chronos/__init__.py

```diff
@@ -153,7 +157,8 @@ def _check(self, resp, content):
             try:
                 payload = json.loads(content)
             except ValueError:
-                self.logger.error("Response not valid json: %s" % content)
+                if expect_json:
+                    self.logger.error("Response not valid json: %s" % content)
```

Comment:
how about using the Content-Type header here rather than requiring a parameter? Chronos seems to set it correctly